src/components/Modal/Modal.e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { browser } from 'protractor';
+import {
+  createStoryUrl,
+  waitForVisibilityOf,
+} from 'wix-ui-test-utils/protractor';
+import { modalTestkitFactory } from '../../testkit/protractor';
+
+describe('modal', () => {
+  const storyUrl = createStoryUrl({
+    kind: 'Tests',
+    story: 'Modal',
+    withExamples: true,
+  });
+  const dataHook = 'e2e-storybook-modal-wrapper';
+
+  beforeEach(() => browser.get(storyUrl));
+
+  it('should render', async () => {
+    const driver = modalTestkitFactory({ dataHook });
+    await waitForVisibilityOf(await driver.element(), 'Cannot find Modal');
+
+    expect(await driver.isModalShowed()).toBe(false);
+    await driver.clickOnOpenButton();
```

Comment:
so instead of using the driver here you would do:
```suggestion
    element(by.css('[data-hook=e2e-storybook-modal-open-btn]')).click();
```

Reply:
and import `element`, and `by` from `protractor`